.github/CODEOWNERS

```
@@ -1 +1 @@
-* @braqzen
+* @nfurfaro
```

```suggestion
* @nfurfaro @braqzen
```

The codeowners is used for notifications for reviews and I want to get notified. I'm the primary reviewer anyway.
Moreover, it goes beyond what the interim role is meant to do anyway so I suggest speaking to Nick about this first before adding him.

---

I'm going to check in with management about any policy/directives regarding codeowners.
From my understanding, either individuals or a team may be assigned as codeowners, but that person/team must have write permissions for the repo.